rocketc/static/js/src/rocketc.js

```diff
@@ -1,22 +1,19 @@
 /* Javascript for RocketChatXBlock. */
 function RocketChatXBlock(runtime, element) {
 
-    function updateCount(result) {
-        $(".count", element).text(result.count);
-    }
+    $(function ($) {
+        /* Here's where you'd do things on page load. */
+    });
 
-    var handlerUrl = runtime.handlerUrl(element, "increment_count");
+    var setDefaultChannel = runtime.handlerUrl(element, "set_default_channel");
 
-    $("p", element).click(function(eventObject) {
+    $("#button", element).click(function(eventObject) {
+        var channel = $("#name", element).val();
+        var data = {channel};
         $.ajax({
             type: "POST",
-            url: handlerUrl,
-            data: JSON.stringify({"hello": "world"}),
-            success: updateCount
+            url: setDefaultChannel,
```

Comment:
If we'are doing requests from frontend, we are going to use the public dns right?

Reply:
I have no idea how works